goal_src/engine/debug/default-menu.gc

```diff
@@ -21,12 +21,16 @@
 
 (define *dm-cam-mode-interpolation* (the-as debug-menu 0))
 
-(defun dm-cam-mode-func ((arg0 uint) (arg1 int))
+(defun dm-cam-mode-func ((arg0 (state camera-slave)) (arg1 int))
   (if (and (= arg1 4) arg0)
-   (send-event *camera* 'change-state arg0 *dm-cam-mode-interpolation*)
+   (begin
+    (format 0 "DEBUG <Vaser>: Changing State Arg 0 '~P' Arg 1 '~P'~%" arg0 arg1)
```

Comment:
probably okay to remove these now?  Also, you can use `(when ..` instead of `(if (begin ..` to save some typing if you want